euctr/templates/about.html

```diff
@@ -154,38 +155,39 @@ <h2 id="inconsistent-data">What does “Inconsistent Data” Mean for a Trial?</
         <p>Currently, we identify the following issues:
 
         <ol>
-          <li>A trial in which all sites are either “completed” or “terminated”
+          <li>A trial in which all sites are either “completed” or “terminated 
```

Comment:
looks like a typo here - I think the `"` needs to be reinstated?

Reply:
Always one...